src/ray/core_worker/task_interface.cc

```diff
@@ -100,64 +98,64 @@ CoreWorkerTaskInterface::CoreWorkerTaskInterface(
                                new CoreWorkerRayletTaskSubmitter(raylet_client)));
 }
 
-Status CoreWorkerTaskInterface::SubmitTask(const RayFunction &function,
-                                           const std::vector<TaskArg> &args,
-                                           const TaskOptions &task_options,
-                                           std::vector<ObjectID> *return_ids) {
+raylet::TaskSpecBuilder CoreWorkerTaskInterface::BuildCommonTaskSpec(
+    const RayFunction &function, const std::vector<TaskArg> &args, uint64_t num_returns,
+    const std::unordered_map<std::string, double> &required_resources,
+    const std::unordered_map<std::string, double> &required_placement_resources,
+    std::vector<ObjectID> *return_ids) {
+  raylet::TaskSpecBuilder builder;
   auto &context = worker_context_;
```

Comment:
Minor, but why not just use `worker_context_` instead of `context`? I think that would be clearer.

Reply:
Fixed.